extensions/azurePublishNew/src/components/AzureProvisionWizard.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import { RecoilRoot } from 'recoil';
+
+import { ProvisionAction } from '../types';
+
+import { ChooseProvisionAction } from './ChooseProvisionActionStep';
+import { CreateResourcesWizard, HandOffToAdminWizard, ImportResourcesWizard } from './provisioningWizards';
+type RootStyleProps = {
+  stepIndex: number;
+};
+const Root = styled.div<RootStyleProps>(({ stepIndex }) => ({
+  height: stepIndex === 1 ? 'unset' : 'calc(100vh - 65px)',
```

Comment:
In the future doing this kind of calculation shouldn't be needed if we use a flex style correctly.